js/data.js

```diff
@@ -200,8 +200,8 @@ var data = {
         dozent: ["Prof. Dr. Christoph Meinel", "Maxim Schnjakin"],
         nameLV: "Cloud und Virtualisierung"
     },
-    constraintbasedprogramming: {
-        kurz: "Constraint-based<br />Programming",
+    constraintbasedprogramming3LP: {
```

Comment:
Please do not change keys of existing courses, this breaks the storage function.